CMakeLists.txt

```diff
@@ -2,11 +2,27 @@
 # Copyright (C) Electronic Arts Inc.  All rights reserved.
 #-------------------------------------------------------------------------------------------
 cmake_minimum_required(VERSION 3.1)
+project(EASTL CXX)
+
+#-------------------------------------------------------------------------------------------
+# Sub-projects
+#-------------------------------------------------------------------------------------------
+add_subdirectory(test/packages/EAAssert)
+add_subdirectory(test/packages/EABase)
+add_subdirectory(test/packages/EAStdC)
+add_subdirectory(test/packages/EAMain)
+add_subdirectory(test/packages/EATest)
+add_subdirectory(test/packages/EAThread)
+
+add_subdirectory(benchmark)
```

Comment:
I'm on the fence about this.  I prefer if people already using EASTL don't get broken by this change and I don't see a compelling reason to change it.